boards/PDM/Test/Src/Test_StateMachine.cpp

```diff
@@ -390,7 +396,9 @@ TEST_F(PdmStateMachineTest, check_24v_acc_voltage_can_signals_in_all_states)
         CANMSGS_PDM_ERRORS__24_V_ACC_VOLTAGE_OUT_OF_RANGE_OVERFLOW_CHOICE);
 }
 
-TEST_F(PdmStateMachineTest, check_aux1_current_can_signals_in_non_init_states)
+TEST_F(
+    PdmStateMachineTest,
+    check_aux1_current_can_signals_in_non_init_states_PDM_11_13)
```

Comment:
This means this test applies to two IDs - 11 and 13

Reply:
I think I'd prefer `PDM_11_to_13`, just to be super explicit?

Reply:
Yeah that's a good point i'll fix that